src/NuGet.Indexing/Extraction/DocumentCreator.cs

```diff
@@ -45,6 +45,8 @@ public static Document CreateDocument(IDictionary<string, string> package)
             AddField(document, "Summary", package, "summary", Field.Index.ANALYZED);
             AddField(document, "Tags", package, "tags", Field.Index.ANALYZED, 2.0f);
             AddField(document, "Authors", package, "authors", Field.Index.ANALYZED);
+            // AddField(document, "PackageTypes", package, "packageTypes", Field.Index.ANALYZED);
```

Comment:
Same as other. Leftover from an older, less flexible implementation. deleted.

---
In reply to: [75551979](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r75551979) [](ancestors = 75551979)